examples/RebootMotionAnalysis.ipynb

```diff
@@ -254,13 +198,13 @@
    "outputs": [],
    "source": [
     "#@title Optional: Uncomment to decorate biomechanics metrics with information from the MLB Stats API and filter based on stats from Stats API\n",
-    "\n",
-    "primary_segment_data_df = decorate_primary_segment_df_with_stats_api(primary_segment_data_df)\n",
     "# Filter the segment data down to a subset of pitches using metrics of your choice\n",
-    "primary_segment_data_df = primary_segment_data_df[primary_segment_data_df[\"start_speed\"] > 90]\n",
-    "\n",
     "# Common Issue:\n",
-    "# Missing element: data integrity issues in parsing. Majority of data is still likely fine."
+    "# Missing element: data integrity issues in parsing. Majority of data is still likely fine.\n",
+    "# Common metrics to filter by are: start_speed, end_speed, spin_rate, spin_direction, zone, pitch_type\n",
+    "\n",
+    "# primary_segment_data_df = decorate_primary_segment_df_with_stats_api(primary_segment_data_df)\n",
+    "# primary_segment_data_df = primary_segment_data_df[primary_segment_data_df[\"start_speed\"] > 90]"
```

Comment:
Can you add a little more detail here regarding how to use this feature? probably don't have to go too crazy since we'll let users tell us how much more they want here - but i think a little more detail would help